drivers/flash/flash_stm32_ospi.c

```diff
@@ -1467,6 +1479,9 @@ static int spi_nor_process_bfp(const struct device *dev,
 		case OSPI_DUAL_MODE:
 			data->write_opcode = SPI_NOR_CMD_PP_1_1_2;
 			break;
+		case OSPI_SPI_MODE:
+			data->write_opcode = SPI_NOR_CMD_PP_4B;
+			break;
```

Comment:
Shall be removed.

It is `SPI_NOR_CMD_PP_4B` only when 4-byte address width is used.
The driver will automatically convert `SPI_NOR_CMD_PP` to `SPI_NOR_CMD_PP_4B` when `four-byte-opcodes` is set in the DTS (see line `1522`).

Reply:
ok